src/EntityComponentManager_TEST.cc

```diff
@@ -2874,9 +2874,9 @@ TEST_P(EntityComponentManagerFixture,
   manager.CreateComponent(jointEntity, components::Name("jointEntity"));
   manager.CreateComponent(jointEntity, components::Joint());
   manager.CreateComponent(jointEntity,
-      components::ParentLinkName(parentLinkEntityName));
+      components::ParentName(parentLinkEntityName));
```

Comment:
I see you're changing the component names here, but i don't see the corresponding changes in `components/ParentLinkName.hh` and `components/ChildLineName.hh`. I'm not sure if we've deprecated components before, but I think we can create a `ParentName`/`ChildName` component and alias it to `ParentLinkName`/`ChildLinkName` with a deprecation attribute.

Reply:
https://github.com/gazebosim/gz-sim/pull/1548/commits/41dd47ddc5a4a102346e19c995f04ca44625a8cf